doc/source/users_guide/index.rst

```diff
@@ -51,110 +50,204 @@ This example shows how you can display a mesh:
 
-.. code:: python
+.. code-block:: python
+
+    from ansys.fluent.visualization import Contour
 
-    temperature_contour_manifold = graphics.Contours["contour-temperature-manifold"]
-    temperature_contour_manifold.field = "temperature"
-    temperature_contour_manifold.surfaces = [
+    cont_surfaces_list = [
         "in1",
         "in2",
         "in3",
         "out1",
         "solid_up:1",
         "solid_up:1:830",
     ]
-    temperature_contour_manifold.display("window-3")
-
-Instantiate a contour object with or without a solver session, using a field name and a list of surfaces, as follows.
-Target is either a Graphics object or a solver session.
+    temperature_contour_manifold = Contour(
+        solver=solver_session,
+        field="temperature",
+        surfaces=cont_surfaces_list,
+    )
+    contour_window = GraphicsWindow()
+    contour_window.add_graphics(temperature_contour_manifold)
+    contour_window.show()
 
-.. code:: python
+Display vector
+~~~~~~~~~~~~~~
 
-    temperature_contour_manifold = Contour(field="temperature",
-                                           surfaces=["in1", "in2", "in3", "out1", "solid_up:1", "solid_up:1:830",])
+.. code-block:: python
 
-    temperature_contour_manifold = Contour(field="temperature",
-                                           surfaces=["in1", "in2", "in3", "out1", "solid_up:1", "solid_up:1:830",],
-                                           solver=solver_session)
+    from ansys.fluent.visualization import Vector
 
-.. code:: python
+    velocity_vector = Vector(
+        solver=solver_session,
+        field="pressure",
+        surfaces=["solid_up:1:830"],
+        scale=2,
+    )
+    vector_window = GraphicsWindow()
+    vector_window.add_graphics(velocity_vector)
+    vector_window.show()
 
-    # Create and render contour object on client side.
-    temperature_contour = temperature_contour_manifold.draw(solver=solver_session, target=Graphics(solver_session))
+Display pathlines
+~~~~~~~~~~~~~~~~~
 
-    # Create and render contour object on server side.
-    temperature_contour = temperature_contour_manifold.draw(solver=solver_session, target=solver_session)
+.. code-block:: python
 
-Display vector
-~~~~~~~~~~~~~~
-This example shows how you can display a vector:
+    from ansys.fluent.visualization import Pathline
 
-.. code:: python
+    pathlines = Pathline(solver=solver_session)
+    pathlines.field = "velocity-magnitude"
+    pathlines.surfaces = ["inlet", "inlet1", "inlet2"]
 
-    velocity_vector = graphics.Vectors["velocity-vector"]
-    velocity_vector.surfaces = ["outlet-plane"]
-    velocity_vector.scale = 1
-    velocity_vector.display("window-4")
+    pathlines_window = GraphicsWindow()
+    pathlines_window.add_graphics(pathlines)
+    pathlines_window.show()
 
 Plot operations
 ---------------
-Examples follow for plot operations that PyFluent-Visualization
-supports.
+Plot operations that PyFluent-Visualization supports:
```

Comment:
```suggestion
Use PyFluent-Visualization plot operations as shown in following code examples.
```
Again, the introduction to the subsequent sections needs to be a complete sentence.